Add Expense Midfi (Custom Split)

stephaniejnc opened the pull request:
- reflect new add expense design/functionality
- change individual expense to change total expense (not greyed out yet)

stephaniejnc committed:
starting midfi

stephaniejnc committed:
comment out split equally amount owed

save for next screen

wrenliang reviewed (changes requested):
Gonna leave this PR for Alan to keep working on the add expense page

yan-alan committed:
Merge branch 'main' into add-expense-midfi

yan-alan committed:
Completed Add Expense Except API Call

- Added figma font to app, use .customFont()
- Added drop down and logic for splitting equally or custom counts

yan-alan committed:
groups font changed

wrenliang reviewed (approved):
Looks good! Thanks Alan
